connector/src/main/scala/com/celonis/kafka/connect/ems/storage/EmsUploader.scala

```diff
@@ -160,7 +160,7 @@ class EmsUploader[F[_]](
     )
     A.delay(
       logger.error(
-        s"Failed to upload the file:$file. Status code:${response.status.show}, Error:${error.msg}",
+        s"Failed to upload the file:$file. Status code:${response.status.show}, Error:${error.msg}. Error: ${throwable.getMessage}",
```

Comment:
Does including `error` in this log line add much information?

```suggestion
        s"Failed to upload the file:$file. Status code:${response.status.show}, Exception: ${throwable.getMessage}",
```

Note: I would also do the same logging we have above, in `genericError(...)`